Kudu.Contracts/Kudu.Contracts.csproj

```diff
@@ -142,6 +142,7 @@
     <Compile Include="Settings\DeploymentSettingsExtension.cs" />
     <Compile Include="Settings\IDeploymentSettingsManager.cs" />
     <Compile Include="Settings\SettingsKeys.cs" />
+    <Compile Include="SiteExtensions\InvalidEndpointException.cs" />
```

Comment:
let me know if it is overkill to create a specific type of exception. was reference from "Deployment" code.